templates/conferences/list.html.twig

```diff
@@ -121,6 +121,7 @@
                     {{ futureConferences|length > 0 ? macro.display_conferences(futureConferences, 'future', activeTab) }}
                     {{ pastConferences|length > 0 ? macro.display_conferences(pastConferences, 'past', activeTab) }}
                 </div>
+            {% include "map.html.twig" %}
```

Comment:
Twig `include` tag is not recommended anymore, [Twig recommends to use the function instead](https://twig.symfony.com/doc/3.x/tags/include.html)